.circleci/config.yml

```diff
@@ -142,15 +111,11 @@ jobs:
       - run:
           name: "Build iOS integration tests"
           command: |
-            bundle exec fastlane update_code_signing
-
             pushd example
 
             # required to init configs that pass environment variables to fastlane later
             flutter build ios --config-only integration_test/all_test.dart \
```

Comment:
@evil159 hmm and is it enough, build still succeeds with the single command? AFAIR previously it was needed to first build with `--config-only` flag and only after that build it actually...

Reply:
@yunikkk this is exactly what is happening now - first we update the config with `flutter build ios --config-only ...` and then build it with `bundle exec fastlane build_examples_tests`

Reply:
The instructions are here https://github.com/flutter/flutter/tree/main/packages/integration_test#ios-device-testing, it mentions config-only but is actually quite confusing whether it is mandatory step...

Reply:
> @yunikkk this is exactly what is happening now - first we update the config with `flutter build ios --config-only ...` and then build it with `bundle exec fastlane build_examples_tests`

I see. Not really sure why that second build was here, probably from the times before fastlane was added